google/cloud/aiplatform/models.py

```diff
@@ -2308,8 +2308,6 @@ def batch_predict(
             gcs_source: Optional[Sequence[str]] = None
```

Comment:
Unsure if batch predict supports wildcards or not.

Reply:
Tested on https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/automl/sdk_automl_video_action_recognition_batch.ipynb and does not support wildcards. Can delete the wildcards description.